CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Thanks a lot for your interest in contributing to **editorconfig-checker**!
+
+## Types of contributions
+
+Contributions to **editorconfig-checker** _include_, but is _not limited to_:
+
+- Report a bug.
+- Suggest a new feature idea.
+- Correct spelling errors, improvements, or additions to documentation files (README, CONTRIBUTING...).
+- Improve structure/format/performance/refactor/tests of the code.
+
+All contributions are welcome from anyone willing to work in good faith with other contributors and the community. No contribution is too small and all contributions are valued.
+
+## Open Development
+
+All work on **editorconfig-checker** happens directly on [GitHub](https://github.com/editorconfig-checker/editorconfig-checker). Both core team members and external contributors send pull requests, which go through the same review process.
+
+## Pull Requests
+
+Pull Requests are the way concrete changes are made to the code and the documentation.
+
+**Please first discuss** the change you wish to make via [issue](https://github.com/editorconfig-checker/editorconfig-checker/issues) before making a change. It might save you time.
+
+If you're adding new features to **editorconfig-checker**, please include tests.
+
+### Review process and requirements to merge
+
+To ensure everyone in the community has the opportunity to share their feedback, ideas, and suggestions, a Pull Request must go through a review process, and must meet the following requirements to be merged:
+
+- **Approval:** At least **2 core team members** must approve the Pull Request.
+- **Open Duration:** The Pull Request must be **open for at least 72 hours**.
+- **CI checks:** The Pull Request must **pass all CI checks**, including tests and build.
+
+**Note:** If a critical bug fix is required, and the Pull Request was approved by at least **3 core team members**, the Pull Request **can be merged before the 72 hours** open duration.
```

Comment:
Feel free to tell me what do you think about these "values". The goal is to make it so that everyone can share their feedbacks, and have the time to do so, without however, blocking the project and the "innovation". We might increase or decrease these requirements.

Reply:
I wonder if it would be better to specify the "open for 72h" as "48h of not requesting changes" - as in "the dust has settled". That way there is no blanket approval for changes that started simple and gained a significant change in the last commit, and leaves enough time for everybody catching the change. But then in the end the core team members have the freedom to extend the time anyway, so it might now be that relevant.